# SPDX-License-Identifier: MIT
# SPDX-FileCopyrightText: pooyanjamshidi

bool charging
---
bool result